packages/vats/index.js

```diff
@@ -1,4 +1,3 @@
-// @ts-check
```

Comment:
I'm curious why this should be taken out.
A comment would be ideal.

Reply:
oh. the tsconfig is opt-out now.

I'm uneasy about making such signals implicit. But I suppose it's better than having the next person forget make a new file and forget to include it.